scss/detail.scss

```diff
@@ -0,0 +1,67 @@
+* {
+  font-family: $font-family;
+}
+
+body {
+  background-color: #F4F4F4;
+}
+
+img {
+  max-width: 100%;
+}
+
+.flex {
+  display: flex;
+}
+
+.book-card {
+  width: 75%;
```

Comment:
I don't usually see these details .. but it's important to keep the code in order 😬 ... so, try to sort it alphabetically